docs/cli.md

```diff
@@ -74,3 +74,34 @@ Into this:
 ```sh
 [2017-04-25T17:32:09.662Z] [32mINFO[39m (24280 on SP2): [36mhello world[39m
 ```
+
+If you logged instance of Error object, Pino adds `"type":"Error"` to json.
```

Comment:
=> "If an instance of `Error` is logged, Pino adds `"type":"Error"` to the logged JSON.

Reply:
fixed